lib/sqlalchemy/dialects/mssql/base.py

```diff
@@ -1488,6 +1488,9 @@ def visit_FLOAT(self, type_, **kw):
         else:
             return "FLOAT(%(precision)s)" % {"precision": precision}
 
+    def visit_DOUBLE(self, type_, **kw):
```

Comment:
for mssql this should be visit_double.  the lower case SQL types (i.e. "Double") are the ones that do translation.  when someone asks for uppercase DOUBLE, they need to get "DOUBLE", and if the DB raises an error that is fine.

Reply:
Thanks for the feedback!

To make sure I understand for this specific instance in MSSQL, instead of replacing `visit_DOUBLE` completely with a lowercase `visit_double` i.e.
```
visit_double(self, type_, **kw):
    return "FLOAT"
```

The approach for this `base.py` file would be to remove this `visit_DOUBLE` function entirely, and any generic `sqltypes.Double` type will compile as a `DOUBLE' in DDL which will indeed raise a MSSQL DB error since `DOUBLE` is not supported, but is fine since they should not be using an unsupported generic type in this instance?

Reply:
yes I think that is correct.

the main thing is if someone uses "DOUBLE", it should be rendering exactly the word "DOUBLE", even though that's not going to work on some backends.    Otherwise what's the point of having "Double" vs. DOUBLE.

Reply:
Great, updated :+1: